libs/ledger-live-common/src/hw/index.ts

```diff
@@ -90,9 +90,14 @@ export const discoverDevices = (
 /**
  * Tries to call `open` on the 1st matching registered transport implementation
  *
+ * An optional timeout can be added. A timeout is applied directly in this function (racing between the matching Transport and the timeout).
+ * But also passed to each `open` Transport implementation. As there is no easy way to abort a Promise (returned by `open`), the Transport will
+ * continue to try connecting to the device even if this function timeout was reached. But certain Transport implementations can also use this
+ * timeout to try to stop the connection attempt.
```

Comment:
maybe make it more explicit how it works: the `timeoutMs` parameter is passed to the `open` method of the transport module so each transport implementation can make use of that parameter and implement the timeout internally.

Reply:
fixed ;)